src/main/java/com/casaculturaqxd/sgec/App.java

```diff
@@ -47,7 +48,7 @@ public static void setRoot(Parent objVisualizacao) throws IOException {
         scene.setRoot(objVisualizacao);
     }
 
-     public static void setRoot(String fxml) throws IOException {
+    public static void setRoot(String fxml) throws IOException {
         lastVisitedPages.add(loadFXML(fxml));
```

Comment:
- [ ] adicionar condição para só adicionar a página na pilha se for diferente da página no topo